glossary.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta name="viewport" content="width=device-width", initial-scale=1.0>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
There were some issues that look like maybe you didn't merge things properly from a previous pull request?